source/CardGames.py

```diff
@@ -76,12 +77,27 @@ def getCard(self):
     self.discarded.append(card)
     return card
 
+def getCard( suit, value):
+  deck = Deck()
+  my_card = Card( suit.capitalize(), value, None, None)
+  for card in deck.cards:
+    if card == my_card:
+      return card
+  return None
+
 class Player:
   def __init__(self, name, money: int = 0):
     self.name = name
     self.hand = []
     self.knownCards = []
     self.money = money
+    self.setsOfFour = {}
+
+  def addSetOfFour(self, value: int):
+    if value in self.setsOfFour:
+      self.setsOfFour[value] += 1
```

Comment:
Looks good. This is a big part of the project, its what makes go fish go fish, great job!

Reply:
Thank you! Go Fish!